main.go

```diff
@@ -470,7 +471,7 @@ func loadConfig() {
 			debug("Disconnected from MQTT (monitoring)")
 		}
 		opts := mqtt.NewClientOptions()
-		opts.AddBroker(fmt.Sprintf("%s://%s:%d", "tcp", getConfig().Monitor.MQTT.Server, getConfig().Monitor.MQTT.Port))
+		opts.AddBroker(fmt.Sprintf("%s://%s:%d", "tcp", getConfig().Monitor.MQTT.Scheme, getConfig().Monitor.MQTT.Server, getConfig().Monitor.MQTT.Port))
```

Comment:
You seem to be missing a variable from the format string here (3 variables in the format string, 4 passed as argument). Nevertheless, based on the [documentation](https://pkg.go.dev/github.com/eclipse/paho.mqtt.golang#ClientOptions.AddBroker), the proper format seems to be the following: `ssl://host:port` for mqtts (or `tcp://host:port` in case of mqtt), are you sure it works with mqtt/mqtts://?

Reply:
Based on the [code](https://github.com/eclipse/paho.mqtt.golang/blob/7b759f1d9ddf721efc39c07c442da116a70c47fd/netconn.go#L40C15-L40C15) it does accept multiple variations, however in order to avoid introducing a new config variable, I would simply use the following:
`opts.AddBroker(fmt.Sprintf("%s:%d", getConfig().Monitor.MQTT.Server, getConfig().Monitor.MQTT.Port))`
This way you can specify the scheme in the server address (e.g. `mqtts://host` or `tcp://host` or` mqtt://host` etc.) and if you don't, standard mqtt (tcp) will be assumed by the mqtt client library. Care to test this?

Reply:
Yeah I missed the `tcp` bit when I forked and did the PR, the line I actually used in the local code I'm using didn't have it (which is why it was working for me I expect).

I've just tested your version, which makes more sense, and it works fine specifying the scheme in the server address:

```go
opts.AddBroker(fmt.Sprintf("%s:%d", getConfig().Monitor.MQTT.Server, getConfig().Monitor.MQTT.Port))
```

![image](https://github.com/a-bali/janitor/assets/88721839/35970a0f-ed5d-4d71-bf18-3076bb2cba4b)

As an aside, `ssl://` does also work, I just wasn't aware you could use `ssl://` instead of `mqtts://`. So I'll leave it up to you (might be worth adding something in the documentation about schemes?).

![image](https://github.com/a-bali/janitor/assets/88721839/abcfb191-ca6c-4280-b29f-a482c50433c3)